spec/unit/validator_spec.rb

```diff
@@ -371,6 +371,18 @@ def podspec_path
         validator.result_type.should == :error
       end
 
+      it 'runs xcodebuild with correct arguments for code signing' do
+        Validator.any_instance.unstub(:xcodebuild)
+        validator = Validator.new(podspec_path, SourcesManager.master.map(&:url))
+        validator.stubs(:check_file_patterns)
+        validator.stubs(:validate_url)
+        validator.expects(:`).with('which xcodebuild').twice.returns('/usr/bin/xcodebuild')
+        command = 'xcodebuild clean build -target Pods CODE_SIGN_IDENTITY=-'
+        validator.expects(:`).with("#{command} 2>&1").once.returns('')
```

Comment:
It is intentional, to both test that iOS builds are using the simulator and that OS X builds do not use the simulator. Otherwise a passing implementation could also be always passing `-sdk iphonesimulator`, which would be wrong, of course.